bionic/deriver.py

```diff
@@ -301,28 +304,28 @@ def register_provider(provider):
         # behavior downstream.
         self._followup_dnode_lists_by_dnode = dict(followup_dnode_lists_by_dnode)
 
-    def _set_up_bootstrap(self):
+    def _set_up_core(self):
```

Comment:
How about using the term `initialize` instead of `set_up` or `finalize`?

Reply:
The reason I prefer `finalize` is that `initialize` suggests the core isn't usable if it's not initialized. (That's how we use "initialize" elsewhere.) In this case, there is a usable core, just not the best one. I'm open to other terms though.